dfhack-config/autonick.txt

```diff
@@ -0,0 +1,13 @@
+# autonick config file
+# every line in this file that is not empty and does
+# not start with "#" will be used as a nickname by the
+# autonick script.
+
+Mouse
```

Comment:
keep in mind that this file will actually get installed to the live dfhack-config/ dir, so if people just run the autonick script, they'll get a dwarf nicknamed "Funny Name for a Dwarf".

The example names in this file should probably be sane defaults (all the nicknames here other than "Funny Name for a Dwarf" are probably perfectly cromulent). or all commented out so autonick does nothing until the file is populated.

Personally I think I vote for a default file of 300 or so reasonable, easy-to-remember and possible-to-differentiate nicknames. Then without the user having to do any custom configuration, they can get some benefit from running autonick.

Reply:
i don't think i have anywhere near 300 names in my own lists, but 30 should be more than achievable.

Reply:
This looks great! Thanks!

there seem to be a few duplicates:
```
$ sort autonick.txt | uniq -d

Mouse
River
```

Reply:
whoops, i missed this comment, terribly sorry! i've removed the duplicates now. very handy trick, that!